### ISSUE TITLE
TextBox and AutoSuggest (might) use deprecated icon codes

### ISSUE THREAD

**Issue by Jay-o-Way** [microsoft/WinUI-Gallery#1093]
### Which version of the app?

WinUI 3 Gallery

### Description

_Coming from issues in PowerToys_

We have issues coming from people with Chinese UI. We used a few deprecated [1] icon codes that would display fine in western languages, but show as "garbled charaters" in Chinese. After updating the Unicode points, these were fixed in our UI. But then there are two more left-overs: the Search icon inside of an \<AutoSuggestBox> and the [X] inside of a \<TextBox>. In the first case we set `QueryIcon="Find"` and in the second case we set no icon. So my conclusion is this is a XAML/WinUI3 issue.

[1] see message at https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font#icon-list

See https://github.com/microsoft/PowerToys/issues/18055#issuecomment-1268074593

### Screenshots

![Image TextBox](https://user-images.githubusercontent.com/68262945/182609355-8a366c93-edf6-4523-8a63-a926fce1ebed.png)
![Image AutoSuggest](https://user-images.githubusercontent.com/68262945/194009493-931b6420-bf41-488e-b270-a33483387ef6.png)

### Windows version

November 2021 Update/21H2 (19044)

**Comment by Jay-o-Way** [comment]
Where can I find the actual Unicode points that are used for these icons?

**Comment by Jay-o-Way** [comment]
Woops, I guess this shouldn't be in the WinUI Gallery repo, but in the WindowsAppSDK repo.

**Comment by chingucoding** [comment]
The actual unicode points can probably be found in the templates of the respective controls. You might want to take a look at the WinUI 2 repository for this since they are open sourced there.

**Comment by Jay-o-Way** [comment]
I can find tons of code and guidelines, but not what I'm looking for. Can you?

**Comment by chingucoding** [comment]
You can find the UWP/WinUI 2 template for the AutoSuggestBox [here](https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/AutoSuggestBox/AutoSuggestBox_themeresources.xaml). In that template GlyphElement that renders has the text `&#xE10A;`.

**Comment by Jay-o-Way** [comment]
Thanks, that confirms my suspicion. Shame on you, Microsoft.

**Comment by chingucoding** [comment]
Since this is an issue with the WinUI control, could you please file an issue on the [WinUI repository]( https://github.com/microsoft/microsoft-ui-xaml) :)

**Comment by Jay-o-Way** [comment]
Fair point. Is it possible that somebody with the right access _moves_ this issue? Or should I create a new copy?

**Comment by chingucoding** [comment]
@gregwoo-microsoft or @bpulliam might be able to move this issue.
